composer.json

```diff
@@ -3,6 +3,27 @@
     "description": "Event-driven, streaming plaintext HTTP and secure HTTPS server for ReactPHP",
     "keywords": ["event-driven", "streaming", "HTTP", "HTTPS", "server", "ReactPHP"],
     "license": "MIT",
+    "authors": [
+        {
+            "name": "Christian Lück",
+            "homepage": "https://clue.engineering/",
+            "email": "christian@clue.engineering"
+        },
+        {
+            "name": "Cees-Jan Kiewiet",
+            "homepage": "https://wyrihaximus.net/",
+            "email": "reactphp@ceesjankiewiet.nl"
+        },
+        {
+            "name": "Jan Sorgalla",
+            "homepage": "https://sorgalla.com/",
+            "email": "jsorgalla@gmail.com"
+        },
+        {
```

Comment:
Looking at my commit history for this repo, I've primary only done peer merges and the odd tooling cleaning. I would not consider myself an author of this repo.

Reply:
@cboden Well the way I look at it, if you are on the team, reviewing code puts you at the same level of responsibility as writing code as by approving the code you say "I trust this code as if I wrote it myself and I take the responsibility for it as if I wrote it myself".

Reply:
🤷‍♂️ ok 😄